--- build.gradle	2020-12-22 18:35:14.000000000 +0800
+++ build.gradles	2020-12-22 18:40:54.000000000 +0800
@@ -319,7 +319,7 @@
     if (branch.startsWith("master")) {
         return "master"
     } else {
-        return branch
+        return "pre-release"
     }
 }
 
@@ -433,7 +433,7 @@
     }
     androidTestImplementation 'androidx.test.uiautomator:uiautomator:2.2.0'
     // For Spoon snapshot, until 2.0.0 is released
-    androidTestImplementation "com.squareup.spoon:spoon-client:2.0.0-SNAPSHOT"
+    androidTestImplementation "com.squareup.spoon:spoon-client:1.7.0"
     // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
